IdentityCore/src/workplacejoin/MSIDPkeyAuthHelper.m

```diff
@@ -146,4 +150,16 @@ + (NSString *)createDeviceAuthResponse:(NSString *)audience
     return [MSIDJWTHelper createSignedJWTforHeader:header payload:payload signingKey:[identity privateKeyRef]];
 }
 
++(void)saveTelemetryForAdfsPkeyAuthChallengeForUrl:(NSURL *)submitUrl
+                                              code:(NSUInteger)code
+                                           context:(id<MSIDRequestContext>)context
+{
+    NSString *submitUrlString = submitUrl.absoluteString;
+    if ([submitUrlString containsString:@"/adfs/ls/"] || [submitUrlString containsString:@"/adfs/oauth2/"])
```

Comment:
I think we have a similar check somewhere in ADFSAuthority utilities. Let's reuse that?

Reply:
Thanks for pointing it out, changed condition to use that method.